lib/smack/SmackRep.cpp

```diff
@@ -1012,13 +1017,47 @@ std::string SmackRep::code(llvm::CallInst& ci) {
 
   std::string s = fmt;
   for (unsigned i=1; i<ci.getNumOperands()-1; i++) {
-    const Expr* a = arg(f, i, ci.getOperand(i));
+    Value* a = ci.getOperand(i);
     std::string::size_type idx = s.find('@');
     assert(idx != std::string::npos && "inline code: too many arguments.");
 
+    llvm::Type* target_type = a->getType();
+    bool is_cast = false;
+    if (idx + 1 < s.length()) {
+      switch(s[idx+1]) {
+      case 'c': {
```

Comment:
LLVM code style does not prescribe opening and closing of curly braces for case statements. Remove those.